Assets/SteamVR_Unity_Toolkit/Scripts/Controls/2D/RadialMenu/RadialMenu.cs

```diff
@@ -0,0 +1,300 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.Events;
+using System.Collections.Generic;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+[ExecuteInEditMode] //Lets us set up buttons from inspector option
+public class RadialMenu : MonoBehaviour {
+
+    #region Variables
+    public List<RadialMenuButton> Buttons;
+	public GameObject buttonPrefab;
```

Comment:
Another tab/space formatting issue, again i guess auto formatting the code in whatever editor would fix it